packages/issuer-rest/lib/oid4vci-api-functions.ts

```diff
@@ -392,11 +394,24 @@ export function pushedAuthorizationEndpoint<DIDDoc extends object>(
   })
 }
 
-export function getMetadataEndpoint<DIDDoc extends object>(router: Router, issuer: VcIssuer<DIDDoc>) {
-  const path = `/.well-known/openid-credential-issuer`
-  router.get(path, (request: Request, response: Response) => {
+export function getMetadataEndpoints<DIDDoc extends object>(router: Router, issuer: VcIssuer<DIDDoc>) {
+  const credentialIssuerHandler = (request: Request, response: Response) => {
     return response.send(issuer.issuerMetadata)
-  })
+  }
+  router.get(WellKnownEndpoints.OPENID4VCI_ISSUER, credentialIssuerHandler)
+
+  const authorizationServerHandler = (request: Request, response: Response) => {
+    return response.send(issuer.authorizationServerMetadata)
+  }
+  router.get(WellKnownEndpoints.OAUTH_AS, authorizationServerHandler)
+  
+  const openidFederationHandler = (request: Request, response: Response) => {
```

Comment:
Why is a REST API implementation of a OID4VC protocol which has little relation to OIDF all of a sudden handling OIDF metadata and wellknowns?